packages/compiler/src/core/checker.ts

```diff
@@ -2397,14 +2414,6 @@ export function createChecker(program: Program): Checker {
     // Evaluate the properties after
     checkModelProperties(node, type.properties, type, mapper);
 
-    for (const prop of walkPropertiesInherited(type)) {
```

Comment:
This doesn't seem to cause any issue to remove. By looking at this logic it wasn't making much sense anyway. We were getting the augmented symbol table for the model node and then for each member setting the symbol type to be that one. 

This is a problem here because the `node.symbol` is always the same for each model instance but then we keep setting a different prop type

Reply:
This code is nonsense, happy to have it gone.